### DISCUSSION TITLE
Private Nuget Package cannot be found

### DISCUSSION THREAD

**Discussion by yithril** [datalust/seq-tickets#1425] [Help]
I've created my own Seq App. I created a .nuspec file, and then set the id of the nuget package myself. I then created a package and copied the package over to a local folder C:/nugetpackages. I then went into SEQ, went to Apps, then selected my feed, copy pasted the package id from the .nuspec file, and it tells me "Could not install app. A package with ID 'EventArchive' was not found in the feed. I set the id myself in the .nuspec file so I know it's correct. What could be wrong?

**Comment by nblumhardt** [comment]
Hi @yithril - does the .NUPKG filename have the form `Id.Version.nupkg`?

**Reply by yithril** [reply]
Ok I retried it with that format. However, now I'm getting "A type marked with [SeqApp] was not found in AdminEventArchive (at: C:\ProgramData\Seq\Packages\nugetfeed-104\AdminEventArchive\1.0.0\lib)"

**Reply by nblumhardt** [reply]
Thanks for your reply!

If there's any way you can share/post a link to the source of your package we might be able to spot the issue, but a better option might be to align your CSPROJ file and build script with one of the examples like https://github.com/datalust/seq-app-htmlemail or https://github.com/datalust/seq-app-opsgenie.

It sounds like you're building the package manually, which will require getting a bunch of details correct that CSPROJ and build scripts in those repositories will already be doing. Let me know if this helps!
